.github/workflows/build.yml

```diff
@@ -90,5 +90,4 @@ jobs:
           target: default
           arch: ${{matrix.arch}}
           tags: |
-            docker.io/hashicorp/${{env.repo}}:${{env.version}}
-            ecr.public.aws/hashicorp/${{env.repo}}:${{env.version}}
```

Comment:
I'm assuming this is being handled elsewhere in the release pipeline now?

Reply:
We've actually never published these images to ECR, and never intended to. Including this line was just a copy paste mistake from the base example.